pandas/core/internals/array_manager.py

```diff
@@ -820,10 +835,11 @@ def take(self, indexer, axis: int = 1, verify: bool = True, convert: bool = True
             new_axis=new_labels, indexer=indexer, axis=axis, allow_dups=True
         )
 
-    def _make_na_array(self, fill_value=None):
+    def _make_na_array(self, fill_value=None, use_na_proxy=False):
+        if use_na_proxy:
+            return NullArrayProxy(self.shape_proper[0])
```

Comment:
if we have both use_na_proxy and a non-None fill_value does that indicate something went wrong?  i.e should we not silently ignore fill_value here?

Reply:
thanks